### PR TITLE
made solution

### PR DESCRIPTION
**Pull request by Sashk52**

### CODE REVIEW

**Review by ShemsheyMaxim — APPROVED** [review]

File: src/main/java/MyRunnableClass.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    private final Counter counter;
+
+    public MyRunnableClass(Counter counter) {
+        this.counter = counter;
+    }
+
+    @Override
+    public void run() {
+        int i = 0;
+        while (i < 100) {
```
Comment (Sofasmile) [inline src/main/java/MyRunnableClass.java]:
```suggestion
        while (counter.getCount() < 100) {
```

File: src/main/java/MyRunnableClass.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    private static final Logger logger = Logger.getLogger(MyRunnableClass.class);
+    private final Counter counter;
+
+    public MyRunnableClass(Counter counter) {
+        this.counter = counter;
+    }
+
+    @Override
+    public void run() {
+        int i = 0;
```
Comment (Sofasmile) [inline src/main/java/MyRunnableClass.java]: this variable is redundant here

**Review by Sofasmile — CHANGES_REQUESTED** [review]

File: src/main/java/MyRunnableClass.java (hunk 1 above)
Comment (Sashk52) [reply src/main/java/MyRunnableClass.java]: done

File: src/main/java/MyRunnableClass.java (hunk 2 above)
Comment (Sashk52) [reply src/main/java/MyRunnableClass.java]: done

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**